config/application.rb

```diff
@@ -112,8 +112,9 @@ def config.database_configuration
       ENV["DISCOURSE_MULTISITE_CONFIG_PATH"] || GlobalSetting.multisite_config_path
     config.multisite_config_path = File.absolute_path(multisite_config_path, Rails.root)
 
+    config.autoload_lib(ignore: %w[common_passwords emoji generators javascripts tasks])
+    Rails.autoloaders.main.do_not_eager_load(config.root.join("lib"))
```

Comment:
I am curious about the change in Rails 7.1 which requires us to do this now. Will you be able to share more? Thank you 🙇

Reply:
It’s what’s recommended here when upgrading: https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#config-autoload-lib-and-config-autoload-lib-once. They say it should still work, but it’s probably better to follow the new convention :slightly_smiling_face: